README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## About me
+
+My name is Yanwen. I am currently a **MAPSS** student in Psychology. 
+Before this course, I have had some experience with:
+* R
+  * ggplot2
```

Comment:
Nice use of indent; perhaps it would be more effective if you included more R-competencies. I'm sure if you've used ggplot you can do basic things like file importing, basic transformations :)